### PR TITLE
Update for hlf-2

### PR DESCRIPTION
**Pull request by Kelvin-M**
Update helm repo add for hlf-2.
Add build config

Run it with `./ci/run-ci.py --substra-chaincode hlf-2 --substra-backend hlf-2 --substra-tests hlf-2 --hlf-k8s hlf-2`

### CODE REVIEW

File: ci/run-ci.py [hunk 1]
```diff
@@ -58,10 +58,11 @@
...
 KEY_SERVICE_ACCOUNT = 'substra-208412-3be0df12d87a.json'
 
-SUBSTRA_TESTS_BRANCH = 'master'
+SUBSTRA_TESTS_BRANCH = 'hlf-2'
 SUBSTRA_BRANCH = 'master'
-SUBSTRA_BACKEND_BRANCH = 'master'
-HLF_K8S_BRANCH = 'master'
+SUBSTRA_BACKEND_BRANCH = 'hlf-2'
+SUBSTRA_CHAINCODE_BRANCH = 'hlf-2'
+HLF_K8S_BRANCH = 'hlf-2'
```
Comment (AurelienGasser) [inline ci/run-ci.py:65]: uh oh

**Review by AurelienGasser — CHANGES_REQUESTED** [review]
I think it would be clearer and more readable if you introduced a `CHAINCODE_COMMIT` global variable, the value of which would be set in `clone_repos`.

Then you don't have to add conditionals and parameters (`chaincode_tag`) in multiple places 

Just a single

```
data['appChannels'][i]['chaincodes'][0]['image']['tag'] = f'ci-CHAINCODE_COMMIT'
```

File: ci/cloudbuild/fabric-peer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+steps:
```
Comment (inalgnu) [inline ci/cloudbuild/fabric-peer.yaml:1]: Correct me if I'm mistaken but it looks like we have similarities between the 3 build files, maybe we should have one config and add a ${DOCKERFILE_IMAGE} variable?

Comment (Kelvin-M) [reply ci/cloudbuild/fabric-peer.yaml]: Done

**Comment by Kelvin-M** [comment]
> I think it would be clearer and more readable if you introduced a `CHAINCODE_COMMIT` global variable, the value of which would be set in `clone_repos`.
> 
> Then you don't have to add conditionals and parameters (`chaincode_tag`) in multiple places
> 
> Just a single
> 
> ```
> data['appChannels'][i]['chaincodes'][0]['image']['tag'] = f'ci-CHAINCODE_COMMIT'
> ```

Done !

**Review by inalgnu — DISMISSED** [review]
LGTM

File: ci/run-ci.py [hunk 3]
```diff
@@ -374,7 +397,8 @@ def build_images(configs):
 def build_image(tag, image, config):
     branch = config["branch"]
     commit = config["commit"]
-    config_file = os.path.join(DIR, f'cloudbuild/{image}.yaml')
+    name = config["name"]
+    config_file = os.path.join(DIR, f'cloudbuild/{name}.yaml')
 
     extra_substitutions = ''
     if image == 'substra-tests':
```
Comment (AurelienGasser) [inline ci/run-ci.py:404]: We could use `name` here?

**Review by AurelienGasser — CHANGES_REQUESTED** [review]
Nice work!

Two more requests:

1. Could you please update the `This script performs the following operations:` section in the doc (at the top of the `run-ci.py` file)? 
2. Have you successfully run the end-to-end tests?

File: ci/run-ci.py (hunk 3 above)
Comment (Kelvin-M) [reply ci/run-ci.py:404]: Done.

**Comment by Kelvin-M** [comment]
Yes, e2e tests successfully passed. Not on your side ?

**Comment by AurelienGasser** [comment]
@Kelvin-M I haven't tried it on my end :)

**Review by AurelienGasser — DISMISSED** [review]

**Comment by AurelienGasser** [comment]
@Kelvin-M I got an error when trying to run this

```
# Setup Helm
+ helm repo add stable https://charts.helm.sh/stable
WARNING: "kubernetes-charts.storage.googleapis.com" is deprecated for "stable" and will be deleted Nov. 13, 2020.
WARNING: You should switch to "https://charts.helm.sh/stable"
Error: repository name (stable) already exists, please specify a different name
FATAL: Command '['helm repo add stable https://charts.helm.sh/stable']' returned non-zero exit status 1.
```

It works when I remove

```
    call('helm repo add stable https://charts.helm.sh/stable')
```

Maybe we need to update helm/skaffold?

**Review by AurelienGasser — APPROVED** [review]

**Pull request merged**